applications/byte_patrol/tests.py

```
@@ -1,2 +1,377 @@
+        )
+
+        # Verify our middleware is in the middleware stack
+        middleware_path = "applications.byte_patrol.middleware.IPRestrictionMiddleware"
+        self.assertIn(
+            middleware_path,
+            settings.MIDDLEWARE,
+            f"The middleware {middleware_path} is not in the middleware stack",
+        )
+
+        # Create a proper get_response function that simulates the next middleware in the chain
+        def get_response(request):
```

Ruff (0.11.9)

Missing return type annotation for private function `get_response`

(ANN202)